### ISSUE TITLE
App geschikt maken voor Atag One Zone

### ISSUE THREAD

**Issue by Boomdijk** [magtimmermans/com.cyclone-software.atagone#10]
Beste Marcel,

ik weet niet hoe ik je op een andere manier moet bereiken, dus maar even zo. Ik wil graag meehelpen om je app geschikt te maken voor de One Zone. Ik ben geen programmeur, maar kan best wel iets doen om het mogelijk te maken om wat meer richting de Zone te komen.

wat kan ik doen?

**Comment by magtimmermans** [comment]
Okay, Ik heb de ATAG One thermostaat gehackt. Helaas is de support van ATAG nul. Dit is wel vrij lastig en toen ik bezig was zag ik ook iemand anders die dit gedaan het. Wat je zou kunnen doen is zoeken of deze ondersteunt worden door andere platformen zoasl Home Assistant of zo. Dan kan ik kijken hoe ze dit gedaan hebben. Het is idee is de communicatie tussen de app van atag en de thermostaat te onderscheppen.

**Comment by WhoopySnoopy** [comment]
De Atag Zone one werkt prima op google assist. Jammer genoeg niet op Homey.  Ik prijs de dag dat deze wens van velen uitkomt met een goed werkende Homey app.

**Comment by simusimu** [comment]
sinds vandaag ben ik een atag one zone plus atag warmtepomp rijker geworden en heb ook intresse, zijn jullie al iets verder gekomen?
john

**Comment by magtimmermans** [comment]
Nee, ik heb zelf geen mogelijkheid om uit te vinden hoe deze werkt/gehackt kan worden. Atag levert nul support indeze.

**Comment by WhoopySnoopy** [comment]
Helaas John is er alleen een app voor de Atag zone maar niet voor de Atag zone one. Maar... via Google assist kun je de Atag zone one wel aansluiten. Met het gesproken commando : "OK Google zet de verwarming op 20 graden" kun je de Atag aansturen.
Enne John. Atag is een goede keuze! Het is een van de koningen onder de verschillende merken. Ik ben benieuwd hoe de warmtepomp bevalt en of je alles met Homey en Google onder controle krijgt. Succes!

**Comment by simusimu** [comment]
bedankt voor jullie reactie, atag zone lijkt meer open te zijn dan one, google assist werkt goed, web cloud connect  misschien icd niet kunnen vinden.
met wireshark eens kijken of ik pakketjes kan ontcijferen
warmtepomp werk goed maar wel bekend traag tov gas
john

**Comment by L00Cyph3r** [comment]
For the users with an Atag One Zone, the one that is using the cloud service (that can use https://www.atagzone.remotethermo.com/), I've been looking for a way to implement it locally, but to no avail so far. I did manage to get it working using their cloud service in HA via this integration: https://github.com/fustom/ariston-remotethermo-home-assistant-v3
It's not ideal, and would love to see a solution to remove the dependency on online services, but at least it works now.

I know it says Ariston, but the underlying portal is RemoteThermo which is compatible with multiple devices according to this issue: https://github.com/fustom/ariston-remotethermo-home-assistant-v3/issues/48

Hope this helps the ones who got the same but different thermostat.
